.github/workflows/validate.yml

```diff
@@ -41,11 +41,15 @@ jobs:
       uses: golangci/golangci-lint-action@537aa1903e5d359d0b27dbc19ddd22c5087f3fbc # v3
       with:
         version: "${{ env.LINT_VERSION }}"
+        only-new-issues: true
```

Comment:
This seems wrong.

Reply:
we have been having issues with the linter including files and even directories not included in any of the PR edits, this is a workaround until we figure it out.